### ISSUE TITLE
Server TRANSFER loses Backups

### ISSUE THREAD

**Issue by Trimilur** [pterodactyl/panel#4770]
### Current Behavior

If you transfer a server the backup is not longer available.

error":"The requested backup was not found on this server.

### Expected Behavior

Backup should be transferred correctly 

### Steps to Reproduce

Transfer server

### Panel Version

1.11.3

### Wings Version

1.11.5

### Games and/or Eggs Affected

SpaceEngineers Default

### Is there an existing issue for this?

- [X] I have searched the existing issues before opening this issue.
- [X] I have provided all relevant details, including the specific game and Docker images I am using if this issue is related to running a server.
- [X] I have checked in the Discord server and believe this is a bug with the software, and not a configuration issue with my specific system.

**Comment by matthewpi** [comment]
Node-local backups are not transferred, they are still on the previous node.  Transfers should probably be prevented from occurring with local backups.

**Comment by potvinp** [comment]
Where do the backups end up moving to once the transfer is complete? Transferred a server to another node and the backups seem to no longer be in `/var/lib/pterodactyl/backups/` on the original node for the server's UUID.

**Comment by theFra985** [comment]
I noticed the same behavior but I was able to find the backups on the source host under the backup directory with the original UUID (note that backup UUIDs are independent from the server's).
Transferring them to the new host seem to restore the backup functionalities from the panel.

PS:
If you need to retrieve the backup UUIDs for a server a way is to inspect the webpage of the backups tab, opening the "Networking" tab of the devtools interface, waiting a bit for a new request to appear (you need to keep the original tab focused for that to happen or you can just switch to another tab and then get back to "Backups"). Once the "backups?page=1" request pops up you just need to click on it and get the UUIDs from the "Response" tab.

An alternative way is to just load `/api/client/servers/8694d80c-c821-4227-9db3-8416870f49b7/backups` replacing the UUID with your server's and get them from there.

**Comment by matthewpi** [comment]
Duplicate of https://github.com/pterodactyl/panel/issues/2623
